test_scripts/API/ButtonSubscription/009_SubscribeButton_INVALID_request_INVALID_DATA.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app requests invalid SubscribeButton request.
+-- SDL does:
+-- 1) respond SubscribeButton(INVALID_DATA) to mobile app
+-- 2) Not send OnHashChange with updated hashId to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variables ]]
+local buttonNamePass = "PRESET_0"
+local invalidButtonName = {
+    "IncorrectName", -- invalid enum value
+    123,             -- invalid type
+    ""               -- empty value
+}
+
+local function rpcIncorStruct(pInvalidButtonName)
```

Comment:
Please use rpcIncorStruct only for invalid struct check, for invalidButtonName please use rpcUnsuccess